lib/compilerdeps.js

```diff
@@ -28,7 +28,3 @@ function $DONE(err) {
   $262.destroy();
 }
 
-function $LOG(str) {
-  print(str);
-}
```

Comment:
I've confirmed that this is not used an any existing test/* or harness/* files in test262